Source/ui_qt/bootablelistdialog.cpp

```diff
@@ -57,7 +60,16 @@ BootableListDialog::BootableListDialog(QWidget* parent)
 	        [&](bool) {
 		        QModelIndex index = ui->listView->selectionModel()->selectedIndexes().at(0);
 		        bootable = model->GetBootable(index);
-		        accept();
+		        if(!m_s3_processing)
+		        {
+			        accept();
+		        }
+		        else
+		        {
+			        QMessageBox::warning(this, "Warning Message",
```

Comment:
Could we move this to a function? Like `displayWarningMessage` or something. I see the code appears 3 times.